storage/kvstore/filesystemstore/tests/UNITTESTS/FileSystemStore/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set(TEST_NAME filesystemstore-unittest)
+
+add_executable(${TEST_NAME})
+
+target_include_directories(${TEST_NAME}
+    PRIVATE
+        ${MBED_PATH}
```

Comment:
This is a problem with LittleFileSystem.h and LittleFileSystem.cpp as it refers to the littflefs headers with the complete path like `storage/filesystem/littlefs/littlefs/lfs_util.h` and fixed the path issue to avoid including the mbed-os into include directories